data-prepper-plugins/prometheus-sink/src/main/java/org/opensearch/dataprepper/plugins/sink/prometheus/configuration/AuthenticationOptions.java

```diff
@@ -8,10 +8,10 @@
 
 public class AuthenticationOptions {
 
-    @JsonProperty("http_basic")
+    @JsonProperty("http-basic")
     private BasicAuthCredentials httpBasic;
 
-    @JsonProperty("bearer_token")
+    @JsonProperty("bearer-token")
```

Comment:
Please change this back to `bearer_token`.

Reply:
@dlvenable Resolved